cirq-core/cirq/protocols/json_serialization_test.py

```diff
@@ -782,6 +782,24 @@ def custom_resolver(name):
     assert_json_roundtrip_works(my_dc, resolvers=[custom_resolver] + cirq.DEFAULT_RESOLVERS)
 
 
+def test_json_serializable_dataclass_no_decorator():
```

Comment:
I'd suggest naming this so it stands on its own, e.g. `test_dataclass_json_dict` (then no change needed if `json_serializable_dataclass` goes away).

Reply:
done